scripts/exwave_prep.sh

```diff
@@ -638,55 +638,55 @@
       chmod 744 cmdfile
 
-    ymdh_rtofs=${PDY}00 # RTOFS runs once daily
-    ymdh_end=`$NDATE ${FHMAX_WAV_CUR} ${ymdh_rtofs}`
-    NDATE_DT=${WAV_CUR_HF_DT}
-    FLGHF='T'
+      ymdh_rtofs=${PDY}00 # RTOFS runs once daily use ${PDY}00
+      ymdh_end=`$NDATE ${FHMAX_WAV_CUR} ${PDY}00`
+      NDATE_DT=${WAV_CUR_HF_DT}
+      FLGHF='T'
 
-    if [ ${CFP_MP:-"NO"} = "YES" ]; then nm=0 ; fi # Counter for MP CFP
-    while [ "$ymdh_rtofs" -le "$ymdh_end" ]
-    do
+      if [ ${CFP_MP:-"NO"} = "YES" ]; then nm=0 ; fi # Counter for MP CFP
+      while [ "$ymdh_rtofs" -le "$ymdh_end" ]
+      do
 # Timing has to be made relative to the single 00z RTOFS cycle for that PDY
-      fhr_rtofs=`${NHOUR} ${ymdh_rtofs} ${PDY}00`
-      fext='f'
+        fhr_rtofs=`${NHOUR} ${ymdh_rtofs} ${PDY}00`
+        fext='f'
 
-      fh3_rtofs=`printf "%03d" "${fhr_rtofs#0}"`
+        fh3_rtofs=`printf "%03d" "${fhr_rtofs#0}"`
 
-      curfile1h=${COMIN_WAV_CUR}/rtofs_glo_2ds_${fext}${fh3_rtofs}_1hrly_prog.nc
-      curfile3h=${COMIN_WAV_CUR}/rtofs_glo_2ds_${fext}${fh3_rtofs}_3hrly_prog.nc
+        curfile1h=${COMIN_WAV_CUR}/rtofs_glo_2ds_${fext}${fh3_rtofs}_1hrly_prog.nc
+        curfile3h=${COMIN_WAV_CUR}/rtofs_glo_2ds_${fext}${fh3_rtofs}_3hrly_prog.nc
 
-      if [ -s ${curfile1h} ]  && [ "${FLGHF}" = "T" ] ; then
-        curfile=${curfile1h}
-      elif [ -s ${curfile3h} ]; then
-        curfile=${curfile3h}
-        FLGHF='F'
-      else
-        echo ' '
-        set $setoff
-        echo ' '
-        echo '************************************** '
-        echo "*** FATAL ERROR: NO CUR FILE $curfile ***  "
-        echo '************************************** '
-        echo ' '
-        set $seton
-        postmsg "$jlogfile" "FATAL ERROR - NO CURRENT FILE (RTOFS)"
-        err=11;export err;${errchk}
-        exit 0
-        echo ' '
-      fi
+        if [ -s ${curfile1h} ]  && [ "${FLGHF}" = "T" ] ; then
+          curfile=${curfile1h}
+        elif [ -s ${curfile3h} ]; then
+          curfile=${curfile3h}
+          FLGHF='F'
+        else
+          echo ' '
+          set $setoff
+          echo ' '
+          echo '************************************** '
+          echo "*** FATAL ERROR: NO CUR FILE $curfile ***  "
+          echo '************************************** '
+          echo ' '
+          set $seton
+          postmsg "$jlogfile" "FATAL ERROR - NO CURRENT FILE (RTOFS)"
+          err=11;export err;${errchk}
+          exit 0
+          echo ' '
+        fi
 
-      if [ ${CFP_MP:-"NO"} = "YES" ]; then
-        echo "$nm $USHwave/wave_prnc_cur.sh $ymdh_rtofs $curfile $fhr_rtofs > cur_$ymdh_rtofs.out 2>&1" >> cmdfile
-        nm=`expr $nm + 1`
-      else
-        echo "$USHwave/wave_prnc_cur.sh $ymdh_rtofs $curfile $fhr_rtofs > cur_$ymdh_rtofs.out 2>&1" >> cmdfile
-      fi
+        if [ ${CFP_MP:-"NO"} = "YES" ]; then
+          echo "$nm $USHwave/wave_prnc_cur.sh $ymdh_rtofs $curfile $fhr_rtofs > cur_$ymdh_rtofs.out 2>&1" >> cmdfile
```

Comment:
again, $nm needs to be included for srun (HERA), but should not be included for mpirun (WCOSS).